android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -156,11 +175,24 @@ private int toKeyCode(long action) {
     }
 
     public static class NotificationService extends Service {
+
         @Override
         public IBinder onBind(Intent intent) {
             return null;
         }
 
+        @Override
+        public void onCreate() {
+            super.onCreate();
+
+            if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                /*Notification notification = new Notification.Builder(MusicControlModule.INSTANCE.context, MusicControlModule.INSTANCE.CHANNEL_ID)
+                        .build();*/
```

Comment:
Thank you! Can you cleanup this unused code?

Reply:
I have cleaned up the unused line, I think you can merge.